### PR TITLE
Writing stuff

### PR DESCRIPTION
**Pull request by sant0s12**
I leave this open as a draft so we know someone is working on it.

Feel free to change the title lol

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,60 @@
...
+
+
 (Add the bibliographic references you intend to use)
 (Explain possible extension to the above models)
 (Code / Projects Reports of the previous year)
 
 
 ## Research Methods
 
+Cellular Automata
```
Comment (NatalieSuter) [inline README.md:56]: You sure?

File: README.md [hunk 2]
```diff
@@ -1,41 +1,60 @@
...
 
 ## The Model
+We model a straight multi-lane highway, where cars enter from the left and exit to the right. To determine the acceleration of each car at a given time step, we use a modified version of the Intelligent Driver Model (IDM) [1]. Since we are modeling a multi-lane highway, we also need to model lane-changing behaviour. This is achieved by utilizing the lane-changing model MOBIL [2].  
+To analyse the results of our simulation, we define our independent variables as time, if a general speed limit is in place and the proportions of different drivers/cars, and our dependent variables as the location of cars and their associated velocity/acceleration at a given time step. The dependent variables can easily be measured by storing the corresponding parameters at every time step in a file.
```
Comment (NatalieSuter) [inline README.md:16]: Actually how do we measure throughput? At the end of the lane?

File: README.md [hunk 3]
```diff
@@ -1,41 +1,60 @@
...
+Highways are an important part of our infrastructure and are used by a lot of people every day for different reasons: To commute to work, transport goods and to go on holidays. Thus a general speed limit may have significant consequences for many people. In particular, we want to analyze the effects of a general speed limit on the throughput of multi-lane highways.  
+While many (European) countries have a general speed limit in place, Germany is one of few countries that doesn't have such a limit and where controversial debates surrounding a general speed limit happen regularly: First article in 1975 (Zeit) "Der alte Streit ist neu entflammt: Tempostopp auf Autobahnen - ja oder nein?", more recently in 2019 (Andreas Scheuer (German Verkehrsminister) gegen neue Tempolimit-Debatte). Both sides seemingly use the same argument: A general speed limit decreases travel time by reducing traffic congestion, while without such a speed limit travel time is decreased because people can drive faster and thus reach their destination quicker. We want to see which of the argumentations hold.
```
Comment (NatalieSuter) [inline README.md:12]: Should we leave the things you cite as refrences or is that too much?

**Review by NatalieSuter — APPROVED** [review]
Pretty good for now I think.

**Comment by dnezam** [comment]
No, I don't think Cellular Automata is correct

**Pull request merged**